arrays/arrays.c

```diff
@@ -105,11 +105,16 @@ void arr_append(Array *arr, char *element) {
 
   // Resize the array if the number of elements is over capacity
   // or throw an error if resize isn't implemented yet.
-
+  
+  if (arr->capacity <= arr->count) {
+      fprintf(stderr, "IndexError: Index is out of range");
```

Comment:
Error should say that the array is at capacity. This error won't be necessary when you finish working on this function because the array will be resized to accommodate more elements when it is at capacity.